docs/docs/sourcing-from-gentics-mesh.md

```diff
@@ -0,0 +1,198 @@
+---
+title: Sourcing from Gentics Mesh
+---
+
+## Using Gentics Mesh
+
+[Gentics Mesh](https://getmesh.io) is an open-source self-hosted headless CMS that supports GraphQL. Contents like posts, products, images and more are represented by elements called "nodes". These nodes can then be queried with the familiar GraphQL syntax.
+
+The model of nodes can be defined within "schemas". In our demo we have schemas for vehicles, images and categories.
+
+## Getting started
+
+In this guide you'll create a complete project capable of querying data from a Gentics Mesh server.
+
+### Start Mesh
+
+Gentics Mesh is a self-hosted CMS. You can run it for example via docker:
+
+```shell
+docker run -p 8080:8080 -d gentics/mesh-demo
+```
+
+The UI can be accessed on http://localhost:8080/mesh-ui/ with default login (admin/admin).
+
+### Install the boilerplate
+
+Now let's create a simple Gatsby starter site.
+
+```shell
+gatsby new gatsby-site https://github.com/gatsbyjs/gatsby-starter-default
+```
+
+Navigate inside of the project with `cd gatsby-site`.
+
+### Add the Source Plugin
+
+Additionally, you need the the `gatsby-source-graphql` library. Because Gentics Mesh uses GraphQL, you will take advantage of Gatsby's ability to simply stitch two GraphQL APIs together, reducing the time required to transform content. There is no need to use a special gatsby-source-x-cms plugin, the GraphQL source plugin is all you need.
```

Comment:
```suggestion
Additionally, you need the `gatsby-source-graphql` plugin. Because Gentics Mesh uses GraphQL, you will take advantage of Gatsby's ability to stitch two GraphQL APIs together, reducing the time required to transform content. There is no need to use a special gatsby-source-x-cms plugin, the GraphQL source plugin is all you need.
```